yellowbrick/regressor/residuals.py

```diff
@@ -670,6 +709,12 @@ def draw(self, y_pred, residuals, train=False, **kwargs):
                 residuals, bins=50, orientation="horizontal", density=True, color=color
             )
 
+        # Add residuals histogram
+        if self.qqplot in {True}:
+            osm, osr = probplot(residuals, dist='norm', fit=False)
```

Comment:
From my understanding probplot generates a probability plot, which should not be confused with a Q-Q or a P-P plot - are(osm,osr) the appropriate values to use for a Q-Q plot?????

Reply:
`scipy.stats.probplot` computes quantiles of sample data and quantiles of specified theoretical distribution, which is exactly what is needed for Q-Q plot. So, the quote from the scipy documentation
> probplot generates a probability plot, which should not be confused with a Q-Q or a P-P plot.

is a bit misleading, because `scipy.stats.probplot` plots exactly Q-Q plot. I'm pretty sure, that I plotted Q-Q plot using `scipy.stats.probplot` correctly.
Here are some links to support my argument: [definition of probability plot](https://en.wikipedia.org/wiki/Probability_plot), [definition of Q-Q plot](https://en.wikipedia.org/wiki/Q%E2%80%93Q_plot), [answer on stackoverflow](https://stackoverflow.com/a/13865874).

Reply:
There is a difference between probplot and Q-Q plot however, scipy's `probplot` actually provides the sample quantiles on the y-axis and Theorectical quantile on the x-axis which is exactly the definition of a Q-Q plot.  Just for our education, Normally, ProbaPlot is the same as a Q-Q plot, however probabilities are shown in the scale of the theoretical distribution (x-axis) and the y-axis contains unscaled quantiles of the sample data.